.github/workflows/rust.yml

```diff
@@ -15,6 +15,17 @@ jobs:
       run:
         working-directory: ./rust
     steps:
+      - uses: actions/cache@v3
```

Comment:
i think you need to change the trigger path so this change will trigger the rust CI jobs?

Reply:
hmm, after doing some research it seems the paths are fine as they are also added the same in the mac-build section, @eddyxu is there any way where I can test it?